assets/custom.js

```diff
@@ -243,4 +243,23 @@ $('.remove').click(function(){
             console.log('error', error);
         }
     })
+});
+
+//Display Footer sub-menu when footer menu is clicked
+$('.footer__links-heading').click(function(){
+    const chevron = $(this).find('.fa-chevron-down');
+    const submenu = $(this).next();
+    const scrollHeight = submenu.prop('scrollHeight');
+    const expandedHeight = scrollHeight + "px";
+    const mediaQuery = window.matchMedia('(max-width: 767px'); //Allows footer menu heading to be clicked ONLY on mobile viewports
+    if (mediaQuery.matches) {
```

Comment:
this is cool. i've never used this. i've always used the jquery way: `$(window).width()` 

ie. (`if ($(window).width() > 767 {}`)

matchMedia seems like a better solution, so i'd stick with it